casper/src/main/resources/RevVault.rho

```diff
@@ -48,6 +48,17 @@ in {
           }
         } |
 
+        contract RevVault(@"unfAuthKey", unf, ret) = {
```

Comment:
let's not use the abbreviation, let's call it `unforgeableNameAuthKey`. `unforgeableAuthKey` can be confusing w.r.t what's unforgeable.

Reply:
Done